src/app/src/pages/redteam/report/components/TestSuites.tsx

```diff
@@ -327,14 +326,7 @@ const TestSuites: React.FC<TestSuitesProps> = ({ evalId, categoryStats, plugins
                         size="small"
                         onClick={() => {
                           const pluginId = subCategory.pluginName;
-                          const categoryType = subCategory.type;
```

Comment:
I couldn't repro a case where `pluginId` was undefined so I got rid of falling back to `categoryType`; unsure whether this is the correct behavior. Thoughts, @sklein12? Otherwise, I'd keep the metric filter when plugin id cannot be used.

Reply:
Got it. I tested it with non-categorical (e.g. `rbac`) and categorical (e.g. `harmful`, `pii`, and `bias`) and it's working, so going to resolve this.